File: KPMG.CRM.Business/Models/BookRoomModel.cs

namespace KPMG.CRM.Business.Models
{
    public  class BookRoomModel
    {
        public string id { get; set; }
        public string name { get; set; }
        public RoomBookModel room { get; set; }
        public string from { get; set; }
        public string to { get; set; }
        public DateTime bookedDate { get; set; }
        public string contactName { get; set; }
    }
    public class RoomBookModel
    {
        public string id { get; set; }
        public string name { get; set; }
    }
}

File: KPMG.BookRoom.Portal/src/app/models/bookroom.model.ts
export interface BookRoomModel{
    id: string,
    name: string,
    bookedDate : Date ,
    from: string,
    to : string,
    room: roomModel
  }

  export interface roomModel{
    id: string,
    name: string,
    code: string,
    isactive: boolean
  }